djstripe/models/base.py

```diff
@@ -144,6 +144,9 @@ def _get_stripe_account_id(self, api_key=None) -> Optional[str]:
         :type stripe_account: string
         """
         api_key = api_key or self.default_api_key
+        # Avoid circular import problems by using the app registry to
+        # get the model class rather than a direct import.
+        account_model = apps.get_model(app_label="djstripe", model_name="account")
```

Comment:
This is nasty. You can't import the account model using a normal import within this function?

Reply:
I didn't try. This is how it was done originally. I simply changed its position as I needed to use it later on too.

Reply:
@jleclanche Updated.